Lib/site.py

```diff
@@ -517,10 +517,7 @@ def register_readline():
         pass
 
     if readline.get_current_history_length() == 0:
-        try:
-            from _pyrepl.main import CAN_USE_PYREPL
-        except ImportError:
-            CAN_USE_PYREPL = False
```

Comment:
I simplify this here because in the current state of the code there is no reason for this import to ever fail.